### PR TITLE
feat(ci): Add github action workflow for develop ios build

### LINKED ISSUE #5722: suite-native: Setup mobile app build for iOS with Github actions
Setup CI build using created fastlane lanes for iOS
use Github CI with MacOS runners.

Related to https://github.com/trezor/trezor-suite/issues/5674

### PR DESCRIPTION
**Pull request by bouchja1**
Closes https://github.com/trezor/trezor-suite/issues/5722

### CODE REVIEW

File: .github/workflows/suite-native_develop_ci.yml [hunk 1]
```diff
@@ -51,3 +51,35 @@ jobs:
...
+      - name: Ruby Setup for Fastlane
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "2.7"
+          bundler-cache: true
+      - name: Install bundler
+        run: gem install bundler
+
+      - name: Fastlane build and deploy
+        env:
```
Comment (bouchja1) [inline .github/workflows/suite-native_develop_ci.yml:78]: @vdovhanych ENVs should be added to github secrets, APPSTORE_KEY_FILE_CONTENT content has to be encoded and put like `base64 appstore_connect_api_key.p8 | pbcopy` and should be enough to run this successfully. At least I hope :D.

**Review by Nodonisko — APPROVED** [review]

**Comment by vdovhanych** [comment]
please wait for me to add env secrets and set up the credentials in Appstore connect.

File: .github/workflows/suite-native_production_ci.yml [hunk 2]
```diff
@@ -39,7 +39,6 @@ jobs:
           SIGNING_KEY_ALIAS: ${{ secrets.SIGNING_KEY_ALIAS }}
           SIGNING_KEY_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD }}
           GOOGLE_PLAY_JSON_KEY_FILE: ./android/firebase_key.json
-          FIREBASE_APP_ID: ${{secrets.ANDROID_FIREBASE_APP_ID}}
```
Comment (vdovhanych) [inline .github/workflows/suite-native_production_ci.yml:42]: we don't need the firebase app id anymore?

**Review by vdovhanych — APPROVED** [review]
Secrets are set up, let's see what happens then :)

File: .github/workflows/suite-native_production_ci.yml (hunk 2 above)
Comment (bouchja1) [reply .github/workflows/suite-native_production_ci.yml:42]: not for prod - it will be distributed with Google Play

**Pull request merged**